index.html

```diff
@@ -1,12 +1,1851 @@
+                                <div class="content">
+                                    <div>
+                                        <span class="category">COVER</span>
+                                        <p class="location">U.S</p>
+                                    </div>
+                                    <h4 class="title">
+                                        'Gleefully Defiant' Trump Spoils for Fight, But Will His Playbook Fail Him?
+                                    </h4>
+                                </div>
+                            </article>
+                        </div>
+                    </div>
+                    <div class="row" style="margin-top: 2rem;">
+
+                        <div class="col-md-3">
+                            <article class="magazine">
+                                <div class="body">
+                                    <img src="https://d.newsweek.com/en/full/1531488/october-11-18-2019.jpg?w=360&h=480&q=90&f=8a6178c02956a944ee55a303cf81ef89"
+                                        alt="magazine" />
+                                </div>
+                                <div class="content">
+                                    <div>
+                                        <span class="category">COVER</span>
+                                        <p class="location">U.S</p>
+                                    </div>
+                                    <h4 class="title">
+                                        'Gleefully Defiant' Trump Spoils for Fight, But Will His Playbook Fail Him?
+                                    </h4>
+                                </div>
+                            </article>
+                        </div>
+                        <div class="col-md-3">
+                            <article class="magazine">
+                                <div class="body">
+                                    <img src="https://d.newsweek.com/en/full/1531488/october-11-18-2019.jpg?w=360&h=480&q=90&f=8a6178c02956a944ee55a303cf81ef89"
+                                        alt="magazine" />
+                                </div>
+                                <div class="content">
+                                    <div>
+                                        <span class="category">COVER</span>
+                                        <p class="location">U.S</p>
+                                    </div>
+                                    <h4 class="title">
+                                        'Gleefully Defiant' Trump Spoils for Fight, But Will His Playbook Fail Him?
+                                    </h4>
+                                </div>
+                            </article>
+                        </div>
+                        <div class="col-md-3">
+                            <article class="magazine">
+                                <div class="body">
+                                    <img src="ht
```

Comment:
remove these comments and any others you have in your code.

Reply:
noted